src/test/java/svnserver/ext/gitea/GiteaIntegrationTest.java

```diff
@@ -168,16 +168,17 @@ private User createUser(@NotNull String username, @NotNull String password) thro
 
   @NotNull
   private User createUser(@NotNull String username, @NotNull String email, @NotNull String password) throws Exception {
+    // Need to create user using command line because users now default to requiring to change password
+    ExecResult createUserHelpResult = gitea.execInContainer("gitea", "admin", "create-user", "--help", "-c", "/data/gitea/conf/app.ini");
+    boolean mustChangePassword = createUserHelpResult.getStdout().indexOf("--must-change-password") > -1;
+    String mustChangePasswordString = mustChangePassword ? "--must-change-password=false" : "";
+    gitea.execInContainer("gitea", "admin", "create-user", "--name", username,
```

Comment:
I think you want to check exit code here to make sure that command completed successfully.